migration/20170713-15-move-library-configuration-to-configurationsettings.py

```diff
@@ -80,15 +80,14 @@
         for k, v in default.items():
             library.default_facet_setting(k).value = v
             
-    # Copy external type regular expression into each collection for each
-    # library.
+    # Copy external type regular expression into each authentication
+    # mechanism for each library.
```

Comment:
The external type regular expression is attached to the library's authentication mechanism, not to the collections. I only found this because it failed in an unexpected way trying to set this value on the Open Access Content Server collection.